app/uk/gov/hmrc/economiccrimelevyregistration/services/DmsService.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyregistration.services
+
+import akka.stream.scaladsl.Source
```

Comment:
Question: Is there a Play alternative to this? I'm just asking because of the future licensing implications of using Akka components directly.

Reply:
@TheDanielDoyle Do you thik this is n issue?

Reply:
This import is brought in by internal-auth-client. It's up to their team to provide an alternative.
@paddynski-opencast @TonyCOgilvie